docs/guides/bridge.md

```diff
@@ -192,7 +185,7 @@ You can check the status of your issue request in the Transactions view in the *
 
 If your Bitcoin transaction has enough confirmations but has not been executed by a Vault yet, click on the issue request that is "Pending". This will open a modal, where you will see a **"Claim KBTC"** (or **"Claim IBTC"**) button. To finalize the Issue process and claim your IBTC/KBTC, either wait for a Vault to auto-execute your request, or click **"Claim"** yourself.
 
-![Tansactions page](../_assets/img/guide/interlay-bridge-issue-claim.png)
+![Tansactions page](../_assets/img/guide/bridge-issue-complete-tx.png)
```

Comment:
Im not sure about this image @nud3l ´. Previously there was a modal with claim ibtc but im not able to reach that. Only this 
![Uploading image.png…]()

Reply:
You can reach it when you send less BTC than you requested. Would be great if you could update that!

Reply:
Done